wikiContent/userNotes.md

```diff
@@ -0,0 +1,54 @@
+/u/lili_misstaipei
+[Abuse Warning]Shit List - by /u/MissPicklesMeow - 2 years ago
+/u/musicalhouses
+[Spam Warning]1:10 content ratio - Offical Warning - by /u/MissPicklesMeow - 2 years ago
+[Spam Watch]Not properly commenting on blogpost - by /u/MissPicklesMeow - 2 years ago
+/u/moisanom
+[Abuse Warning]Affil Links & 1:10 Ratio - by /u/MissPicklesMeow - 2 years ago
+/u/bmasterb
+[Abuse Warning]Business Post 1 - by /u/MissPicklesMeow - 2 years ago
+/u/celerythedog
+[Abuse Warning]1:10 rules 1 - by /u/MissPicklesMeow - 2 years ago
+/u/TheBlackTaj
+[Spam Watch]7/7 Watch for 1:10 ratio - by /u/SnowWhiteandthePear - 2 years ago
+/u/MiddleearthFirefly
+[Spam Warning]Delete Posts & Reposts for Karma - Warned - by /u/MissPicklesMeow - 2 years ago
+/u/misschul
+[Spam Warning]Blog Spammer. -stufstuf - by /u/stufstuf - 2 years ago
+/u/Eletas
+[Spam Watch]Hmm, too much content too quickly. - by /u/stufstuf - 2 years ago
+[Good Contributor]Consistently posts quality content. - by /u/stufstuf - 2 years ago
+/u/scarlettma
+[Spam Watch]Possible WT shill/employee - by /u/SnowWhiteandthePear - 2 years ago
+/u/TheBeautyWitches
+[Spam Watch] 1:10 content ratio - warned 3/30, 2/18, 2/17 - by /u/kstoops2conquer - about a year ago
+/u/soleseoulsoul
+[Spam Watch]Unidentified business posts 4/13 & 4/22 - ban warned - by /u/kstoops2conquer - about a year ago
+/u/size6
+[Abuse Warning]Warned4/23 and 4/28 - by /u/kstoops2conquer - about a year ago
+/u/thesnailofitall
+[Spam Watch]1:10 ratio - watch out for lack of participation - by /u/starward- - 11 months ago
+/u/riquelll
+Habitual daily help thread rule violator - by /u/kstoops2conquer - 8 months ago
+/u/sydneyunnie
+[Spam Watch]Possible K-Beauty Now spammer - by /u/Ronrinesu - 7 months ago
+/u/katiesbeautyview
+[Spam Watch]1:10 Ratio - by /u/petitoignon - 2 months ago
+/u/oklettucehead
+[Spam Watch]Potential Meme/Karma Spam - by /u/petitoignon - 2 months ago
+/u/professionalwitch
+[Spam Watch]Potential Meme/Karma Spam - by /u/petitoignon - 2 months ago
+/u/vmnoelleg
+[Spam Watch]Potential Meme/Karma Spam - by /u/petitoignon - 2 months ago
+/u/floralandfading
+[Spam Watch]Beautysesh shill!? - by /u/Ronrinesu - about a month ago
+/u/fairest-one-of-all
+[Spam Watch]new account; questionable results https://www.reddit.com/r/AsianBeauty/comments/66z213/fotd_acne_scars_after_2_months_of_15_step_korean/ - by /u/petitoignon - 25 days ago
+/u/DubuTutorials
+[Spam Watch]soliciting free business advice - by /u/petitoignon - 11 days ago
+/u/ModestNewbie
+Problematically reposted a bloggers non AB related content after the initial post was removed. - by /u/petitoignon - 10 days ago
+/u/Nina_in_ABwonderland
+[Spam Watch]tracking - by /u/petitoignon - a day ago
+/u/just_say_knope
+[Spam Watch]Possible Massdrop shill - by /u/Ronrinesu - about 11 hours ago

```

Comment:
This whole section @ltnerap - none of the notes are rude, but they should probably remain private.